lib/peeps.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+class Peeps
+  
+  attr_reader :id, :peep, :time, :users
+
+  def initialize(id:, peep:, time:)
+    @id = id 
+    @peep = peep
+    @time = time
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      PG.connect(dbname: 'chitter_test')
+    else
+      PG.connect(dbname: 'chitter')
+    end
+    result = con.exec("SELECT * FROM peeps;")
+    result.map do |peep|
+      Peeps.new(id: peep['id'], peep: peep['peep'], time: peep['time'])
+    end
+  end
+
+  def self.create
```

Comment:
In the controller, the Peeps.create method takes two parameters. 

`Peeps.create(peep: params[:peep], time: Time.now.strftime("%m-%d-%Y %H:%M:%S"))`

So definition of the method needs two.